src/quantum/TP6.jsx

```diff
@@ -70,8 +70,19 @@ TP6.propTypes = {
   location: PropTypes.shape({
     search: PropTypes.string.isRequired,
   }).isRequired,
+  platform: PropTypes.string.isRequired,
 };
 
+function getPlatformOptions() {
+  const results = selectFrom(PLATFORMS)
+    .where({ platform: ['win32', 'win64', 'aarch64', 'linux64'] })
+    .select({ id: 'platform', label: 'platform' })
+    .toArray();
+  const unique = new Set(results.map(item => item.label));
+
+  return Array.from(unique).map(item => ({ id: item, label: item }));
+}
```

Comment:
All this can be reduced to 

    return ['win32', 'win64', 'aarch64', 'linux64'].map(v => ({ id: v, label: v }));